lib/mirage.ml

```diff
@@ -16,13 +16,13 @@
  * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  *)
 
-open Functoria
+open Functoria.DSL
```

Comment:
```suggestion
```

according to OCaml-CI, this is unused

Reply:
Oh right, we do `include Functoria.DSL` a bit further down.